Cargo.toml

```diff
@@ -70,7 +71,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 snafu = { version = "0.7", features = ["backtraces"] }
 sqlparser = "0.30"
-tempfile = "3"
+tempfile = "3.4"
```

Comment:
```suggestion
tempfile = "3"
```

Reply:
sure, will make an adjustment.
I've noticed some of the dependencies have `major.minor`, so I put it like that. Out of curiosity, why "3" is enough?
Is it because it's test dependency and the bump up is a bit relaxed?

Reply:
> Is it because it's test dependency and the bump up is a bit relaxed?

This is also a benefit. We don't rely on a specific patch, so we could use the simplest version, as recommended in the style guide.

> Use the simplest version possible (i.e., use the smallest version number that you know is correct, avoid point or patch versions if possible, avoid use of semver ‘operator’s if possible).

https://pingcap.github.io/style-guide/rust/modules.html

We don't have strict restrictions about which version should use, so "major.minor" is ok. Since most crates specify the minor version in their README and we just copy-paste it to our Cargo.toml 🤣